tests/tests_suite_test.go

```diff
@@ -30,7 +30,8 @@ import (
 
 func TestTests(t *testing.T) {
 	RegisterFailHandler(Fail)
-	RunSpecs(t, "Tests Suite")
+	polarionReporter := tests.NewPolarionReporter("polarion.xml")
```

Comment:
I would add flag where you can specify path to output file

Reply:
fixed